### PR TITLE
[logging] Keep important fields from being lost

### PR DESCRIPTION
**Pull request by gregnazario**
Due to the limit on the number of bytes in a log line, we are losing
important data due to some really large fields.  This will keep some of
the important error data around for debugging.  This also raises the max
size of a log line.

Realistically, this does not solve the problem of large logs, or keeping information that some logs may find useful or not useful.  But, as a temporary fix, we will at least keep all errors.

### CODE REVIEW

**Comment by andll** [comment]
The unit test on this PR failed - we can try to land it with bors, but i am not sure if failure was transient

**Comment by gregnazario** [comment]
> The unit test on this PR failed - we can try to land it with bors, but i am not sure if failure was transient

```
thread 'block_storage::block_store::block_store_and_lec_recovery_test::test_executor_restart' panicked at '
    Unable to build all workspace binaries. Cannot continue running tests.
```
This doesn't seem like it's due to the PR.

File: common/logger/src/struct_log.rs [hunk 1]
```diff
@@ -66,7 +69,7 @@ pub struct StructuredLogEntry {
     /// name of the event
     #[serde(skip_serializing_if = "Option::is_none")]
     name: Option<&'static str>,
-    /// rust module (e.g. consensus::round_manager)
+    /// rust module (e.g. consensus::round_manager):156
```
Comment (mimoo) [inline common/logger/src/struct_log.rs:72]: ?

Comment (gregnazario) [reply common/logger/src/struct_log.rs:72]: definitely an accident

**Pull request closed without merging**